src/app/jewelry/jewelry.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Jewelry } from '../jewelry/jewelry.model';
+import { Injectable, EventEmitter } from '@angular/core';
+
+@Injectable()
+export class JewelryService {
+  jewelrySelected = new EventEmitter<Jewelry>();
```

Comment:
It may be better to use a Subject here (possibly a Behavior Subject). 
`selectedJewelry: BehaviorSubject<Jewelry>()`. Let me know if you need clarification on that, but if you use a subject, then consumers can just subscribe to the selected jewery item, and when it changes, the service can just call `this.selectedJewelry.next(selectedItem)` which will notify the subscribers of a change that they can then act on.

Reply:
This is actually a good example of usage:
https://medium.com/@weswhite/angular-behaviorsubject-service-60485ef064fc (if you can't open that, open it in an chrome incognito window which should bypass the paywall)